dist/css/other_assets.css

```diff
@@ -5,3 +5,7 @@
     padding: 0;
     background: url('/assets/original/equipment-dude.jpg') no-repeat;
 }
+
```

Comment:
I would try to use semantic-ui for this rather than custom css.

If there are any custom styling required, it would be good to look into elm-css for that.